src/lib/classes/NetworkManager.ts

```diff
@@ -48,11 +48,8 @@ export class NetworkStore implements Writable<NetworkState> {
 	}
 
 	addConnection(input: { source: Node; target: Node }) {
-		if (input.source === input.target) {
-			return;
-		}
+		if (input.source === input.target) return;
```

Comment:
I personally don't like this, and think we should agree on a standard. But I can tolerate it